opencog/learning/PatternMiner/PatternMinerDistributedWorker.cc

```diff
@@ -505,12 +505,12 @@ bool DistributedPatternMiner::sendRequest(http_request &request, http_response &
             return false;
         }
     }
-    catch (http_exception e)
+    catch (http_exception &e)
```

Comment:
I'm surprised that your tools do not recommend a `const` here ...

Reply:
They usually don't recommend `const`s, but I should've spotted that myself, true.